endpoints/get_conversation.py

```diff
@@ -0,0 +1,27 @@
+import json 
+from models.conversations import Conversation 
+from globals import tracer
+from globals import app
+from session import session
+
+@tracer.capture_method
+def get_conversation(conversation_id):
+
+    # Get a Conversation object from the database
+    with session:
+        conversation = session.query(Conversation).filter_by(id=conversation_id).first()
+
+        if conversation is None:
+            return {
+                'statusCode': 404,
+                'body': json.dumps({'message': f'Conversation with ID {conversation_id} not found.'})
+            }
+        return {"statusCode": 200, "body": json.dumps(conversation.to_dict())}
```

Comment:
Are we not grabbing the text stored in s3 when we call get_conversation? At what point are we accessing that to send as part of the response for this API call?

Reply:
Good call out. The branch wasn't updated. I just updated this branch to reflect the code that's in the lambda function. Now you can see where the data from s3 is being accessed and returned.